Cognite.Simulator.Tests/IntegrationTests/ConnectorRuntimeTest.cs

```diff
@@ -100,7 +100,10 @@ public async Task WriteConfig()
             using var provider = services.BuildServiceProvider();
             TestCdfClient = provider.GetRequiredService<Client>();
 
-            await SeedData.GetOrCreateSimulator(TestCdfClient, SeedData.SimulatorCreate);
+            // if the following call is not made before starting the test, we get an exception in the first test
```

Comment:
what kind of exception?

Reply:
System.ObjectDisposedException : Cannot access a disposed object.
Object name: 'IServiceProvider'.